File: go/ManifoldApi/GetMe.go
package ManifoldApi

import (
	"encoding/json"
	"fmt"
)

func GetMe() User {
	sb := callManifoldApi("GET", fmt.Sprintf("v0/me"), nil)

	var response User
	json.Unmarshal([]byte(sb), &response)
	return response
}

File: src/api/getMe.ts
import { callManifoldApi } from "./callManifoldApi.ts";
import { User, UserSchema } from "./getUser.ts";

export async function getMe(): Promise<User> {
  const sb = await callManifoldApi("GET", "v0/me");
  return UserSchema.parse(JSON.parse(sb));
}
